.github/workflows/ci.yml

```diff
@@ -29,15 +29,15 @@ jobs:
 
     strategy:
       matrix:
-        os: [ubuntu-22.04, macos-13] # TODO: Add windows-2019
+        os: [ubuntu-latest, macos-latest]
```

Comment:
Dont we need linter tests on windows?

Reply:
I enabled it but turned it off, it had random invalid formatting complaint. I forget what the solution to it is, didn't think it'd be necessary if we are running the tests on windows now